.github/workflows/flutter.yml

```diff
@@ -10,7 +10,7 @@ on:
     branches: [ "main" ]
     paths: ['network_tools_flutter/**']
   pull_request:
-    branches: [ "main", "dev" ]
+    branches: [ "main", "dev-flutter" ]
```

Comment:
We are not removing dev so there is no reason to remove it from here

Reply:
I want to run flutter based checks on dev-flutter and dart based checks on dev

Reply:
And also if you check logs for GHA steps we can no more send resource intensive objects in message passing to isolates. Check below error.

Invalid argument(s): Illegal argument in isolate message: object is unsendable - Library:'dart:async' Class: _Future@4048458 (see restrictions listed at `SendPort.send()` documentation for more information)

Reply:
> 

Ho ok, so this is ci file for flutter only